csrc/moe/marlin_moe_wna16/ops.cu

```diff
@@ -192,33 +192,34 @@ int get_scales_cache_size(thread_config_t const& th_config, int prob_m,
 
   if (cache_scales_chunk) {
     int load_groups =
-        tb_groups * pipe_stages * 2;     // Chunk size is 2x pipeline over dim K
-    load_groups = max(load_groups, 32);  // We load at least 32 scale groups
+        tb_groups * pipe_stages * 2;  // Chunk size is 2x pipeline over dim K
+    load_groups = max(load_groups, 32 + 1);  // We load at least 32 scale groups
```

Comment:
Why the +1?

Reply:
I miscalculated the shared memory size occupied by scale. Indeed, there's no need to add 1 here. Updated.